lpc55_isp/src/bin/cfpa-update.rs

```diff
@@ -56,19 +55,11 @@ fn main() -> Result<()> {
 
     let mut rkth = RKTHRevoke::new();
 
-    rkth.rotk0 = ROTKeyStatus::Enabled.into();
+    rkth.enable_keys(true, false, false, false);
 
     cfpa.update_rkth_revoke(rkth)?;
 
-    let mut updated = cfpa.pack()?;
-
-    // need to recalculate sha over the updated data
-    let mut sha = sha2::Sha256::new();
-    sha.update(&updated[..0x1e0]);
-
-    let updated_sha = sha.finalize();
-
-    updated[0x1e0..].clone_from_slice(updated_sha.as_slice());
```

Comment:
Turns out this isn't actually necessary and the ROM will take care of doing the SHA for us